services/utils.ts

```diff
@@ -16,12 +16,16 @@ export function unwrapOrThrowError<T>(
   response: AxiosResponse<ResponseEnvelope<T>>,
   ignoreResponse = false,
 ): T {
-  if (response.data.isOK) {
-    if (ignoreResponse) {
-      return {} as T;
+  try {
```

Comment:
`response.data` does not contain the interface fields so this unwrapper doesn't work

Reply:
This needs to be changed in the near future 🙃